<launch>
    <node pkg="cgn" name="cgn_server" type="generate_grasps_server_local.py" required="true" output="screen"/>
</launch>
